### PR TITLE
Improve spark setup notes and clean up notebooks

### PR DESCRIPTION
**Pull request by changhiskhan**
e.g., remove reference to Label in coco notebook

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,36 +137,51 @@ pip install -e . # pip install -e .[all] to install all optional extras (see "In
...
 ```
 
-Please note that the above sample assumes your local Apache Spark comes with Hadoop 2.7. If you
-installed another version of Hadoop, please use a matching hadoop-aws jar version.
-
-As with other Spark options, there are multiple ways to specify them.
-Please see [Spark documentation](https://spark.apache.org/docs/latest/configuration.html) for
-details.
+If you want to read/write data from/to S3, you will need additional setup:
+
```
Comment (eddyxu) [inline README.md:154]: feel that it is too crowd in `README.md` tho. should we have a ``docs/`` section just for AWS/S3 setup?

No urgent.

File: README.md [hunk 2]
```diff
@@ -137,36 +137,51 @@ pip install -e . # pip install -e .[all] to install all optional extras (see "In
...
+   ```python
+   spark = (
+       SparkSession
+       .builder
+       .appName('rikai-quickstart')
+       .config('spark.jars.packages', 'ai.eto:rikai:0.0.1')
+       .config("spark.driver.extraJavaOptions", "-Dcom.amazonaws.services.s3.enableV4=true")
+       .config("spark.executor.extraJavaOptions", "-Dcom.amazonaws.services.s3.enableV4=true")
+       .config("spark.hadoop.fs.s3a.endpoint", "s3-us-west-1.amazonaws.com") # only for hadoop 2.7x
```
Comment (eddyxu) [inline README.md:170]: IIRC, these are not necessary for every region?

File: README.md (hunk 1 above)
Comment (changhiskhan) [reply README.md:154]: yeah good idea. It started out simple but really snowballed. I'll move it to docs/ section, along with the databricks stuff.

File: README.md (hunk 2 above)
Comment (changhiskhan) [reply README.md:170]: yeah, only for the ones that support v2 or smth i forgot.

**Review by eddyxu — APPROVED** [review]
thanks for working on the doc! LGTM

**Pull request merged**